src/item.cpp

```diff
@@ -1991,6 +1991,30 @@ std::string item::info( std::vector<iteminfo> &info, const iteminfo_query *parts
         if( type->can_use( "MA_MANUAL" ) && parts->test( iteminfo_parts::BOOK_SUMMARY ) ) {
             info.push_back( iteminfo( "BOOK",
                                       _( "Some sort of <info>martial arts training manual</info>." ) ) );
+
+            if( g->u.has_identified( typeId() ) ) {
+                const matype_id style_to_learn = martial_art_learned_from( *type );
+                std::string diff;
+                if( style_to_learn->learn_difficulty <= 2 ) {
+                    diff = _( "easy" );
+                } else if( style_to_learn->learn_difficulty <= 4 ) {
+                    diff = _( "moderately hard" );
+                } else if( style_to_learn->learn_difficulty <= 6 ) {
+                    diff = _( "hard" );
+                } else if( style_to_learn->learn_difficulty <= 8 ) {
+                    diff = _( "very hard" );
+                } else {
+                    diff = _( "extremely hard" );
+                }
+
+                info.push_back( iteminfo( "BOOK",
+                                          string_format( _( "You can learn <info>%s</info> style from it." ), style_to_learn->name ) ) );
+                info.push_back( iteminfo( "BOOK",
+                                          string_format( _( "This fighting style is <info>%s</info> to learn." ), diff ) ) );
+                info.push_back( iteminfo( "BOOK",
+                                          string_format( _( "It'd be easier to master if you'd have skill expertise in <info>%s</info>." ),
+                                                  style_to_learn->primary_skill->name() ) ) );
+            }
```

Comment:
This whole chunk is largely a duplicate of the code in `avatar.cpp` above.  I appreciate it's not *exactly* the same, but at least the first part converting a difficulty number to a string should probably be factored out into a function somewhere.